mappings/net/minecraft/item/Items.mapping

```diff
@@ -241,6 +241,8 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_28043 DRIPSTONE_BLOCK Lnet/minecraft/class_1792;
 	FIELD field_28101 SCULK_SENSOR Lnet/minecraft/class_1792;
 	FIELD field_28409 GLOW_LICHEN Lnet/minecraft/class_1792;
+	FIELD field_28653 MOSS_CARPET Lnet/minecraft/class_1792;
+	FIELD field_28654 MOSS_BLOCK Lnet/minecraft/class_1792;
```

Comment:
Is the item id `moss_block` and the block it `moss`? If yes guess it's just another lilypad or cauldron

Reply:
No, the block and item identifiers are the same. The sound events just use `moss`, though.

Reply:
Hmm, If they are both `moss_block` I'd suggest naming the block field `MOSS_BLOCK` as well, given how `GOLD_BLOCK` etc are named

Reply:
Hm? It is named `Blocks#MOSS_BLOCKS`; the `MOSS` in this pull request is a `BlockSoundGroup` containing sound events such as `block.moss.break`.